apps/src/gamelab/AnimationTab/ItemLoopToggle.jsx

```diff
@@ -0,0 +1,59 @@
+/** @file controls below an animation looping toggle */
+'use strict';
```

Comment:
`'use strict';` isn't actually needed under webpack... it's just everywhere because we've never cleaned this up.  You can omit it.